repos/dir.targets

```diff
@@ -191,6 +196,7 @@
   <Target Name="CreateCombinedRestoreSourceAndVersionProps"
           BeforeTargets="Build">
     <ItemGroup>
+      <_DotNetRestoreSources Include="$(ExtraRestoreSourcePath)" Condition="'$(ExtraRestoreSourcePath)' != ''"/>
```

Comment:
I can't quite tell why this doesn't have the `'$(OfflineBuild)' == 'true'` condition, why is this different from the nuget.config injection?

Reply:
It doesn't need to be offline.  I removed that from the nuget.config injection as well.